test/arithmetic-test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Read the Testing requirements in Lab 1 carefully. The tests should also test for what happens if the arguments are not numbers, which your arithmetic functions don't have any handling for.